packages/vscode-apollo/package.json

```diff
@@ -40,7 +40,7 @@
   ],
   "icon": "images/icon-apollo-teal-400x400.png",
   "activationEvents": [
-    "workspaceContains:**/apollo.config.js"
+    "workspaceContains:**/package.json"
```

Comment:
I'm not sure about this. This would try to load the extension on _any_ project with a package.json, even ones we don't care about.

Is there any other way to know if a project is _supposed_ to be a graphql project, other than with the inclusion of an Apollo config?

Reply:
I decided this was necessary in order to launch the extension and "listen" for the addition of an apollo config if none exists. This way, a project can go from no config -> config w/o reloading VSCode, and expect the extension to pick up on the changes and initialize the project.

I'm open to other ideas here!

Reply:
Is it possible for the error where there' no config to be a toast-style notification that disappears after a few seconds then? This one you have to dismiss I believe right?